public/modules/articles/controllers/articles.client.controller.js

```diff
@@ -4,19 +4,23 @@ angular.module('articles').controller('ArticlesController', ['$scope', '$statePa
 	function($scope, $stateParams, $location, Authentication, Articles) {
 		$scope.authentication = Authentication;
 
-		$scope.create = function() {
-			var article = new Articles({
-				title: this.title,
-				content: this.content
-			});
-			article.$save(function(response) {
-				$location.path('articles/' + response._id);
+		$scope.create = function(isValid) {
+			if (isValid) {
```

Comment:
Seems like a good idea to check, but it would probably make more sense to set the inputs as required and not have to do a check in JS if necessary. Thoughts?